### DISCUSSION TITLE
Holy Shit ！

### DISCUSSION THREAD

**Discussion by Xunhaoz** [1101-GS4719-C/Python-Community-GS4719-C#1354] [Ideas]
**請你注意「排泄」，要為「運動導致剩餘」的過程尋一個合適的動詞，「排擠」、「代謝」其實都可以，但只有「排泄」才能恰如其分地說明狀況。

我們要為「金錢」給出一個發生現象學層面的定義：金錢是一坨佔據了上帝位置的排泄物，簡稱神聖的屎，“Holy shit”。

這個充滿了矛盾張力的定義，能相當準確地反映出人們對金錢的糾結態度。

如果我們對「金錢」這個範疇做隱喻原型分析，就會發現上帝和排泄物是金錢最主要的兩個隱喻源頭。 「當金錢站起來說話時，所有的真理都沉默了——當上帝現身，一切質疑都將消散」「金錢不是萬能的（上帝），沒有金錢是萬萬不能的——金錢雖然不是上帝，但它勝似上帝」，「有錢了不起啊——把自己成全能神了？」；排泄物隱喻更是無處不在：「視金錢如糞土」，「我不要“臭”錢」「一身銅“臭”味」，「被金錢污染了/弄髒了心靈」「這個人貪「污」，「腐敗」」。

金錢這個矛盾體，一面具有神聖律令性，一面又充滿了多餘且污穢的特質。很多人存不下錢來，難以抑制地想要消費，一方面的確是因為物慾，有些東西被營銷得無比誘人，但也有一種人，只要有錢，就一定想辦法找能買的商品，哪怕這個東西本身毫無吸引力，但就是要把錢花掉，不花就是不爽，彷彿肚子有一泡屎不拉就會憋死。

我們迫不及待地想要處理掉那些積淤的髒東西，也不可抑制想要搞清楚那謎一樣的「不止於此」到底是什麼，如此，塑造我們消費觀的三角結構出現了。一頭是「不知道為什麼，反正就想花出去」的貨幣（貨幣端-排泄本能），另一頭是「不知道為什麼，反正一定有道理」的價格（需求端-慾望機制），毫無道理的買賣一拍即合，兩者合力的結果就是「不知道為什麼，我就是想要這個東西」（商品端-戀物癖&拜物教）。

我們被商品耍的團團轉，卻始終甘之如飴，像只快樂的泰迪，這是為何？因為商品既是問題也是答案，既是手段也是目的，人們對商品的信仰具有一種回指性的邏輯完備，就像教徒說「我不知道為什麼相信神，但神能告訴我為什麼」，我們面對商品的態度也是「只要得到了它，我就知道我為什麼想要它了」。

真正勾引我們的，不是商品神秘性背後的真相，而是神秘性本身，是永遠無法實現的“那點意思”——一個女生看見另一個女生背著LV包包，以為那個女生get到了那點意思，心嚮往之，事實上那女生啥也沒get到，她也只是以為另另一個女生get到了而已。 bb以此類推，回指邏輯才永不落空。

在社會關係中，他人的答案充當了我們的答案，我們的答案也被另一些他人的當成自己的答案，最終，一個“中介性”的虛假答案充實了所有人：有沒有那點意思不重要，他人覺得我懂那點意思就夠了——我的幸福感，一總是來自「我認為他人認為我會有幸福感」，他人的幸福感來自他認為我認為他很幸福。這個中介性的虛假答案，在眾人的想像中，成為了一個理想的他人，一個完美的幻象，正是這個從來沒有存在過的幻象激發、牽引著我們的消費慾望（品牌代言人制，就是利用該原理的商業操作——品牌自己說不清的“那點意思”，讓有“那點意思”的人來說）。

這種「我認為他認為，他認為我認為」的間接錯亂想像，必須在無限的商品序列中才能被接續（慾望流→商品流），一旦我們被嵌入這個序列，人與人的關係就在物與物的關係中凝固了，這就是所謂的物化。貨幣的無限增殖也只有在慾望牽引之下的商品的無限序列之中才能實現（貨幣流），我們看到的這根高聳於雲的GPD增長線，單憑技術進步是解釋不了的，它必須是慾望流、商品流、貨幣流“三流匯合”的結果。

這才是所謂商品拜物教和愛欲經濟學的底層原理，如此，商品繁榮和流通現象才能獲得一個更徹底的解釋。如果各種關於戀物癖和消費心理的理論都不能解釋你的病態的消費欲，可以試試這個說法。

關於消費主義的話題我們下期還會再談到。

梳理一下今天的內容：節目一開始，我們問了一個“少年覺得很重要，大人覺得很荒謬”的問題，「錢是什麼」，這個問題引導我們追究價值的真正來源。從重商主義到古典主義，從對古典主義的批判到馬克思的政治經濟學，我們發現這個問題並不好回答。

有兩組觀念的逆轉與綜合需要重視。 1）價值首先是一種信仰，然後才是理性，最終，信仰在自否定的運動中獲得了自身的合法性——通過聲稱我不是宗教而成為貨真價實的宗教；2）金錢首先是精神現象，然後才是經濟現象，如此，經濟社會才表現為一種被「抽象真實統治的具體幻象」——就像語言秩序結構了精神，光學秩序結構我們眼前的世界那般自然而隱秘。

我們試圖用謎一樣視覺與語言系統來類比謎一樣的經濟系統，以尋求啟發，最終得到了一組命題，上半句是「中介退隱，萬物確認自身，相對而立」，下半句則是「萬物流通，差異生成剩餘，“不止於此”」，我們它用來概括價值生成的神秘機制。本期節目輾轉騰挪，機關算盡，也不過是為了論證這組命題的合理性。以它們為前提，我們替很多經濟學中的老概念找到了發生現象學層面的新定義：

價值是因為沒人知道它是什麼，所以人人都覺得它重要的東西；商品是掩蓋了差異的虛假範疇（虛假一般性）；利潤是這些虛假範疇在流通中排泄出來的東西；資本家就是創造這種流通並攫取排泄物的人；品牌是商品世界的黑話；金錢是神聖的屎

當然，得到這些定義還不算完，《資本與少年》這個主題下還有太多問題沒有得到回答，下期，我們要帶著這些觀點，用一個全新的視角，重新理解近一百年的經濟史，繼續給這個荒謬世界一個與之般配的說法。

**Comment by 710388** [comment]
這是什麼神奇的文章

**Comment by NcuJerryKao** [comment]
金錢，這個充滿矛盾張力的概念，確實在人類心中佔據著特殊的位置。你的隱喻分析非常有趣，將金錢比作「神聖的屎」，形象且令人深思。

你提到的「不知道為什麼，我就是想要這個東西」，以及商品、需求和貨幣之間的三角結構，深刻地揭示了我們對金錢的矛盾態度。金錢既是手段，也是目的，既是神秘，又是現實。這種回指性的邏輯，讓我們在無限的商品序列中追求著「不止於此」。

你的觀點也呼應了「拜物教」和「愛欲經濟學」的底層原理，以及金錢與時間、信用、文明之間的關聯。金錢的真正意義，或許不僅僅是經濟單位，更是我們對自由、創造力和幸福的追求。

期待下期節目，重新理解經濟史，為這個荒謬世界找到更加般配的說法。🌟

**Comment by OHoln** [comment]
這篇文章為經濟提出了一個很獨到的觀點，我只想說，我看不懂，但我大受震撼。

**Comment by poiu6fi** [comment]
知識以奇怪的方式進到我的腦子裡
